extensions/samples/assets/shared/scripts/provisionComposer.js

```diff
@@ -451,7 +453,7 @@ const create = async (
     });
 
     // create the app registration
-    const appCreated = await createApp(graphClient, name, appPassword);
+    const appCreated = await createApp(graphClient, botName, appPassword);
```

Comment:
@GeoffCoxMSFT I caught a possible bug here, where we are creating an app registration if no app ID is provided. I haven't noticed that in my previous commit, because as you can see in the issue description, I was providing already existing app ID-s to the script when testing this fix. Now that is fixed as well and taking this into account, I think `botName` as the variable name is more descriptive for the whole function.